src/main/java/io/qdrant/client/QdrantClient.java

```diff
@@ -2204,6 +2207,65 @@ public ListenableFuture<List<BatchResult>> recommendBatchAsync(
 			MoreExecutors.directExecutor());
 	}
 
+	/**
+	 * Performs a batch update of points.
+	 * 
+	 * @param collectionName The name of the collection.
+	 * @param operations The list of point update operations.
+	 * 
+	 * @return a new instance of {@link ListenableFuture}
+	 */
+	public ListenableFuture<UpdateBatchResponse> batchUpdateAsync(String collectionName, List<PointsUpdateOperation> operations) {
+		return batchUpdateAsync(collectionName, operations, null, null, null);
+	}
+
+	/**
+	 * Performs a batch update of points.
+	 * 
+	 * @param collectionName The name of the collection.
+	 * @param operations The list of point update operations.
+	 * @param wait Whether to wait until the changes have been applied. Defaults to <code>true</code>.
+	 * @param ordering Write ordering guarantees.
+	 * @param timeout The timeout for the call.
+	 * 
+	 * @return a new instance of {@link ListenableFuture}
+	 */
+	public ListenableFuture<UpdateBatchResponse> batchUpdateAsync(
+		String collectionName,
+		List<PointsUpdateOperation> operations, 
+		@Nullable Boolean wait, 
+		@Nullable WriteOrdering ordering,
+		@Nullable Duration timeout) {
+
+		UpdateBatchPoints.Builder requestBuilder = UpdateBatchPoints.newBuilder()
+				.setCollectionName(collectionName)
+				.addAllOperations(operations)
+				.setWait(wait == null || wait);
+
+		if (ordering != null) {
+			requestBuilder.setOrdering(ordering);
+		}
+		return batchUpdateAsync(requestBuilder.build(), timeout);
+	}
+
+
+	/**
+	 * Performs a batch update of points.
+	 * 
+	 * @param request The update batch request.
+	 * @param timeout The timeout for the call.
+	 * 
+	 * @return a new instance of {@link ListenableFuture}
+	 */
+	public ListenableFuture<UpdateBatchResponse> batchUpdateAsync(UpdateBatchPoints request, @Nullable Duration timeout) {
```

Comment:
```suggestion
	public ListenableFuture<List<UpdateResult>> batchUpdateAsync(UpdateBatchPoints request, @Nullable Duration timeout) {
```

Would align with the convention of other batch APIs